### DISCUSSION TITLE
How to figure out error messages,similar to the demo for send-transaction

### DISCUSSION THREAD

**Discussion by itsanishjain** [wevm/wagmi#2105] [Q&A]
Hey how to show those error messages when we use send-transaction

![image](https://user-images.githubusercontent.com/43033153/229078148-6a7c565e-cea3-4259-9714-1d9b312accc5.png)

https://wagmi.sh/examples/send-transaction

**Comment by sjunaiduk** [comment]
Hi,

I would make use of the prepare hook which not only returns a config object but also an error object. 

You can then conditionally render that error message if it exists
